components/table/Table.tsx

```diff
@@ -42,6 +42,8 @@ export { ColumnsType, TablePaginationConfig };
 
 const EMPTY_LIST: any[] = [];
 
+export type TableAction = 'paginate' | 'sort' | 'filter';
```

Comment:
https://github.com/ant-design/ant-design/blob/fd0038e9e05e31350eff13e015f3250525a64f89/components/grid/row.tsx#L12

Reply:
@AdrianoRuberto Please imitate this.

Reply:
couldn't it be an ENUM?

`export enum TableActions {'paginate', 'sort', 'filter'};`

Reply:
just did as @afc163 pointed